src/middlewared/middlewared/plugins/virt/instance.py

```diff
@@ -309,6 +309,9 @@ async def image_choices(self, data):
                             'archs': [v['arch']],
                             'variant': v['variant'],
                             'instance_types': list(instance_types),
+                            'secureboot': (
```

Comment:
We probably need to be able to differentiate between three states:
- `Secure boot is required`
- `Secure boot is not required`
- `Secure boot is required to be off`.

Reply:
Hmm dont think secure boot being required is a possibility

Reply:
That option is not available upstream, only thing we can have is whether secureboot needs to be off or not.
`secureboot` false here means secureboot is not supported, but I will add another state true/false/none